common/source/docs/common-cabling-guide.rst

```diff
@@ -0,0 +1,275 @@
+  - Splitting devices across multiple I2C buses.
+  - `Employing I2C bus accelerators or extenders <https://docs.thunderfly.cz/avionics/TFI2CEXT01/>`_, to boost signal quality over extended cable lengths.
+
+I2C Bus Accelerators and Translators
+++++++++++++++++++++++++++++++++++++
+
+- **I2C Bus Accelerators** (e.g., `ThunderFly TFI2CEXT01 <https://docs.thunderfly.cz/avionics/TFI2CEXT01/>`_) physically divide the bus into segments, amplifying signals and reducing the impact of wiring capacitance, thus improving signal integrity and reliability on longer cables. The TFI2CEXT01 can also perform voltage level translation between 3.3V and 5V logic levels.
+- **I2C Address Translators** (e.g., `ThunderFly TFI2CADT01 <https://docs.thunderfly.cz/avionics/TFI2CADT01/>`_) allow multiple identical devices with the same I2C address to be connected by remapping device addresses dynamically, resolving conflicts and simplifying sensor integration.
+
+CAN
+---
+
+:ref:`CAN <common-canbus-setup-advanced>` bus is provided on many autopilots for use with :ref:`DroneCAN <common-uavcan-setup-advanced>` peripherals. ESCs, GPS/Compass, Rangefinders, and many other peripherals and sensors are being added to the list of available DroneCAN devices everyday. CAN provides a robust method of communicating with peripherals with data integrity, even with long leads.
+
+
+Signals
++++++++
+
+Power and ground lines are typically provided alongside the CAN signals on standard 4-pin connectors (e.g., JST-GH).
+
+.. list-table:: CAN Signal Colour Examples
+   :header-rows: 1
+   :widths: 15 20 20 20 20
+
+   * - Signal
+     - Pixhawk
+     - ThunderFly
+     - CUAV
+     - Zubax
+   * - +5V
+     - Red
+     - Red
+     - Red
+     - Red
+   * - CAN_H
+     - Black
+     - White
+     - White
+     - White
+   * - CAN_L
+     - Black
+     - Yellow
+     - Yellow
+     - Yellow
+   * - GND
+     - Black
+     - Black
+     - Black
+     - Black
+
+Cable Recommendations
++++++++++++++++++++++
+
+CAN cables should use twisted pairs to reduce electromagnetic interference (EMI) and maintain signal integrity:
+
+- Twist each signal pair (CAN_H/CAN_L and +5V/GND) approximately 10 turns per 30 cm.
+- Twist both pairs together about 4 turns per 30 cm.
+- Maintain separation from high-power and high-noise cables.
+
+Due to its differential signaling and robust protocol, CAN is particularly suitable for applications requiring reliability over longer cable lengths and in electrically noisy environments.
+
+SPI
+---
+
+SPI (Serial Peripheral Interface) is a synchronous serial communication protocol used by autopilots to connect higher-bandwidth peripherals and sensors, such as optical flow sensors, specialized telemetry modems, barometers, IMUs, and other advanced digital sensors. It supports full-duplex communication using separate lines for data input and output. Most autopilots have processors with multiple SPI ports. 
+
+Signals
```

Comment:
again same comemnt

Reply:
Understood.  Something the general color guide doesn’t capture. I could clarify that further in the text if it's helpful.